pkg/admission/handlers/sts.go

```diff
@@ -0,0 +1,56 @@
+package handlers
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/kanopy-platform/hedgetrimmer/pkg/admission"
+	"github.com/kanopy-platform/hedgetrimmer/pkg/limitrange"
+	appsv1 "k8s.io/api/apps/v1"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+	kadmission "sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
+type StatefulSetHandler struct {
+	DefaultDecoderInjector
+	ptm admission.PodTemplateSpecMutator
+}
+
+func NewStatefulSetHandler(ptm admission.PodTemplateSpecMutator) *StatefulSetHandler {
+	return &StatefulSetHandler{ptm: ptm}
+}
+
+func (sts *StatefulSetHandler) Kind() string { return "StatefulSet" }
+
+func (sts *StatefulSetHandler) Handle(ctx context.Context, req kadmission.Request) kadmission.Response {
+	log := log.FromContext(ctx)
+
+	lrConfig, err := limitrange.MemoryConfigFromContext(ctx)
+	if err != nil {
+		reason := fmt.Sprintf("invalid LimitRange config for namespace: %s", req.Namespace)
+		log.Error(err, reason)
+		return kadmission.Denied(reason)
+	}
+
+	in := &appsv1.StatefulSet{}
```

Comment:
nit: This may be a clearer name. since `req.Object.Raw` is technically `in`.

```suggestion
	out := &appsv1.StatefulSet{}
```